### DISCUSSION TITLE
contabo object storage

### DISCUSSION THREAD

**Discussion by rasul-chopurov** [pocketbase/pocketbase#4336] [Q&A]
how to use contabo storage?
i am add and this error
<img width="261" alt="Screenshot 2024-02-12 at 19 18 25" src="https://github.com/pocketbase/pocketbase/assets/13812501/1f64f7b6-21d8-4030-b147-c1eba66e225c">

**Comment by ganigeorgiev** [comment]
I'm not familiar with Contabo and their S3 offering.

The 403 error usually indicates that something is wrong with the credentials so double check:
- that there are no typos or extra spaces with the client id and secret
- if Contabo supports ARN/IAM or other similar policy controls, review their configuration

**Comment by jasonhornsby** [comment]
I was facing the same issue and it turned out I was using the wrong endpoint. Make sure you do not have your bucket name in the endpoint URL. The contabo UI gives you the wrong endpoint with the bucket name attached.
